presto-docs/src/main/sphinx/connector/system.rst

```diff
@@ -34,6 +34,11 @@ System Connector Tables
 
 The catalogs table contains the list of available catalogs.
 
+``metadata.table_properties``
+^^^^^^^^^^^^^^^^^^^^
+
+The table_properties table contains the list of available table properties.
```

Comment:
The table properties table contains the list of available table properties that can be set for tables in a connector.